src/amo/components/CollectionManager/index.js

```diff
@@ -326,12 +333,19 @@ export class CollectionManagerBase extends React.Component<
   }
 }
 
-export const extractId = (ownProps: Props) => {
+export const extractId = (ownProps: Props): string => {
   const { collection } = ownProps;
   return `collection-${collection ? collection.slug : ''}`;
 };
 
-export const mapStateToProps = (state: AppState) => {
+export const mapStateToProps = (
+  state: AppState,
+): {|
+  clientApp: null | string,
+  currentUserId: null | number,
```

Comment:
We had a `UserId` type at some point, maybe we could use that.

Reply:
Yes, thanks for pointing that out. I've gone through and replaces all places where we're using `number` for `userId` with `UserId`.